### DISCUSSION TITLE
Use https for accessing kiali GUI from external system

### DISCUSSION THREAD

**Discussion by shinderupesh** [kiali/kiali#5416] [General]
I have setup kiali in k3s env using [helm charts](https://kiali.io/docs/installation/installation-guide/install-with-helm/). After kiali pod is coming in running state I have attached the Loadbalancer using following command:
  `kubectl patch svc kiali -n istio-system -p '{"spec": {"type": "LoadBalancer"}}'` 

I am able to see external port for kiali:
> kubectl get svc -n istio-system kiali
>kiali                  LoadBalancer   10.43.60.50     <pending>       20001:30603/TCP,9090:32749/TCP 

After this I am able to access kiali externally: http://{IP}:30603/kiali

But as specified in [docs](https://kiali.io/docs/installation/quick-start/), I am not able to access kiali using HTTPS.

Is there anything I missing?

**Comment by israel-hdez** [comment]
You probably may also want to read the short page about Accessing Kiali: https://kiali.io/docs/installation/installation-guide/accessing-kiali/

In general, we cannot provide instructions showing how to expose Kiali that would work for everybody. For your specific case, it fully depends on how Load Balancers work in your cloud. Without knowing which cloud you are using, the best I can suggest is to find and read the relevant documentation for Load Balancers of your cloud provider. Some Load Balancers allow you to setup secure access.

If your Load Balancer does not offer a way to secure traffic, you can still configure your certificates directly in Kiali. Currently, these configs does not have a dedicated doc page, but they are mentioned in the Kiali CR reference here: https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.identity
